protos/local/temporal/sdk/core/activity_task/activity_task.proto

```diff
@@ -27,16 +28,16 @@ message Start {
     // The workflow's type name or function identifier
     string workflow_type = 2;
     // The workflow execution which requested this activity
-    common.WorkflowExecution workflow_execution = 3;
+    temporal.api.common.v1.WorkflowExecution workflow_execution = 3;
     // The activity's ID
     string activity_id = 4;
     // The activity's type name or function identifier
     string activity_type = 5;
-    map<string, common.Payload> header_fields = 6;
+    map<string, temporal.api.common.v1.Payload> header_fields = 6;
```

Comment:
No big benefit, but could use `temporal.api.common.v1.Header` here and in other header places

Reply:
All these comments, I actually prefer them without the wrappers, which are _really_ unergonomic in Rust and at least slightly annoying in other languages. You now have an extra layer of optionality that doesn't need to be there.